testdata/book/cookie.yml

```diff
@@ -0,0 +1,38 @@
+desc: Inherit cookies Example
+runners:
+  req:
+    endpoint: ${HTTPBIN_END_POINT:-https://httpbin.org/}
+    openapi3: https://tryapisproxy.com/spec/httpbin
+vars:
+  cookie:
+    key: "cookie-name"
+    value: "cookie-value"
+debug: ${DEBUG:-true}
+steps:
+  bindUuid:
+    desc: UUIDs are numbered in the image of a session.
+    bind:
+      uuid: faker.UUID()
+  setCookie:
+    desc: Returns a set of response headers from the query string.
+    req:
+      /response-headers?Set-Cookie={{ vars.cookie.key }}%3D{{ uuid }}:
+        get:
+          body: null
+    test: |
+      current.res.status == 200
+      && current.res.headers["Set-Cookie"][0] startsWith vars.cookie.key + "="
+  bindCookies:  
+    bind:
+      cookies: steps.setCookie.res.headers["Set-Cookie"]
+  inheritCookie:
+    desc: Returns cookie data.
+    req:
+      /cookies:
+        get:
+          headers:
+            Cookie: "{{ cookies[0] }};"
```

Comment:
The cookie header must be generated by interpreting the contents of the Set-Cookie and taking into account the case where a path or "expire" is specified.